macos/FluentUI/AvatarView/AvatarView.swift

```diff
@@ -436,34 +474,39 @@ fileprivate extension Unicode.Scalar {
 
 fileprivate extension NSColor {
 	/// the text color of the text in the initials view
-	///
-	/// - note: this value doesn't change for dark mode by design as our default background color table
-	/// only provides colors designed to be used with white text, even on dark mode
-    static let initialsViewTextColor: NSColor = .white
 
 	/// the table of background colors for the initials views
-	static let avatarBackgroundColors: [NSColor] = [
-		Colors.Palette.cyanBlue10.color,
-		Colors.Palette.red10.color,
-		Colors.Palette.magenta20.color,
-		Colors.Palette.green10.color,
-		Colors.Palette.magentaPink10.color,
-		Colors.Palette.cyanBlue20.color,
-		Colors.Palette.orange20.color,
-		Colors.Palette.cyan20.color,
-		Colors.Palette.orangeYellow20.color,
-		Colors.Palette.red20.color,
-		Colors.Palette.blue10.color,
-		Colors.Palette.magenta10.color,
-		Colors.Palette.gray40.color,
-		Colors.Palette.green20.color,
-		Colors.Palette.blueMagenta20.color,
-		Colors.Palette.pinkRed10.color,
-		Colors.Palette.gray30.color,
-		Colors.Palette.blueMagenta30.color,
-		Colors.Palette.gray20.color,
-		Colors.Palette.cyan30.color,
-		Colors.Palette.orange30.color
+	static let avatarColors: [DynamicColor] = [
```

Comment:
I think the ColorSet class is still useful here to encapsulate a DynamicColor
 tuple of background/foreground.